routes/cube_routes.js

```diff
@@ -3125,6 +3129,36 @@ router.delete('/format/remove/:cubeid/:index', ensureAuth, param('index').toInt(
   }
 });
 
+router.post('/:id/defaultdraftformat/:formatId', ensureAuth, async (req, res) => {
```

Comment:
for api error handling, please follow the pattern we have used for other api routes using `util.wrapAsyncApi`